tutorials/28_Structured_Output_With_OpenAI.ipynb

```diff
@@ -0,0 +1,591 @@
+{
```

Comment:
<p>Where can we find info on the schema format? Add a link if possible.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepset-ai/haystack-tutorials/pull/412/discussion/#comment-2581346719'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>